srcpkgs/wlroots/template

```diff
@@ -24,9 +25,7 @@ post_install() {
 }
 
 wlroots-devel_package() {
-	depends="MesaLib-devel elogind-devel eudev-libudev-devel libcap-devel
-	 libdrm-devel libinput-devel libxkbcommon-devel pixman-devel wayland-devel
-	 wayland-protocols wlroots-${version}_${revision}"
+	depends="${sourcepkg}-${version}_${revision} ${_devel_depends}"
```

Comment:
Better to use `>=` rather than a specific version.

Reply:
I was wondering, thanks.